samples/msgext-doc-search-csharp/infra/botRegistration/azurebot.bicep

```diff
@@ -0,0 +1,37 @@
+@maxLength(20)
+@minLength(4)
+@description('Used to generate names for all resources in this file')
+param resourceBaseName string
+
+@maxLength(42)
+param botDisplayName string
+
+param botServiceName string = resourceBaseName
+param botServiceSku string = 'F0'
+param botAadAppClientId string
+param botAppDomain string
+
+// Register your web service as a bot with the Bot Framework
+resource botService 'Microsoft.BotService/botServices@2021-03-01' = {
+  kind: 'azurebot'
+  location: 'global'
+  name: botServiceName
+  properties: {
+    displayName: botDisplayName
+    endpoint: 'https://${botAppDomain}/api/messages'
+    msaAppId: botAadAppClientId
+  }
+  sku: {
+    name: botServiceSku
+  }
+}
+
```

Comment:
We should include the `m365extensions` channel in this file which is needed for the plugin to work with Copilot when it is deployed to Azure.